src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -15,20 +19,24 @@
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.IOException;
+import java.nio.file.*;
+import java.security.NoSuchAlgorithmException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.function.BiConsumer;
+import java.util.stream.Collectors;
 
 public class GenotypeGVCFsIntegrationTest extends CommandLineProgramTest {
 
-    //This value was originally 30 but was changed to 10.  We're setting most tests to use 30 to temporarily avoid having
-    //to update the expected outputs
-    private static final List<String> STAND_CALL_CONF_30 = Arrays.asList("-stand_call_conf", "30");
+    private static final List<String> NO_EXTRA_ARGS = Collections.emptyList();
+    private static final String b38_reference_20_21 =largeFileTestDir + "Homo_sapiens_assembly38.20.21.fasta";
```

Comment:
Hmm, ideally this would be a constant in `BaseTest` in public. Can we check this reference into public as well (if it's not already), and add the constant to `BaseTest`? (not a blocker for merging this PR though)